plugins/kotlin/uast/uast-kotlin-fir/tests/testData/legacyRenderLog/ReceiverFun.render.txt

```diff
@@ -1,8 +1,8 @@
 public final class ReceiverFunKt {
-    public static final fun foo(@MyReceiverAnnotation @org.jetbrains.annotations.NotNull $this$foo: java.lang.String) : int {
+    public static final fun foo(@org.jetbrains.annotations.NotNull @MyReceiverAnnotation $this$foo: java.lang.String) : int {
```

Comment:
`KotlinReceiverUParameter` had its own annotation creation which put the annotation on the receiver ahead of nullity, while everything else uses the same logic---computing nullity annotation first. Now, with the same logic, which is based on LC's use-site computation, the order is changed, but aligned with other kinds of variables.